spec/unit/resources/splunk_service_spec.rb

```diff
@@ -1,137 +1,146 @@
 require_relative '../spec_helper'
 
-# TODO: Change how we're stubbing methods during refactoring so we can move this out of the global object space.
 include CernerSplunk::ServiceHelpers, CernerSplunk::RestartHelpers
```

Comment:
Any update here?  One of the goals of the refactor was to not have to include these in the global space.